tests/test_main.py

```diff
@@ -179,6 +181,59 @@ async def test_authenticate_exceptions() -> None:
     assert str(excinfo.value) == "Timeout error: Request timed out"
 
 
+@pytest.mark.parametrize(
+    "fixture_file,method,expected_url,expected_redactions",
+    (
+        (
+            "me.json",
+            "get_homeowner",
+            ME_URL,
+            [
+                '"email": "**redacted**"',
+                '"firstName": "**redacted**"',
+                '"lastName": "**redacted**"',
+                '"deviceSerialNumbers": [\n        "**redacted**"\n    ],\n',
+            ],
+        ),
+        (
+            "notifications.json",
+            "get_notifications",
+            NOTIFICATIONS_URL,
+            [
+                '"serialNumber": "**redacted**"',
```

Comment:
Added this for consistency, now that there is a fixture for `/kem/api/v3/notifications`